Codigo/modSendData.bas

```diff
@@ -68,6 +68,18 @@ Public Enum SendTarget
     ToGroupButIndex
 End Enum
 
+Public Sub SendToConnection(ByVal ConnectionId, Optional Args As Variant)
```

Comment:
remove `Args` as it's not used in the sub ?

Reply:
this is weird but a standard on senddata,  sendto functions has a parameter so we can call the preprareXXX in the same line, its weird here but then its clear in all the calls are like

Sendto(Target, PrepareXXX)

otherwise we would have to split all sendto call in two lines, 1 to preprare the message and anocher to send it (and it will be harder to track de connection between both statements, making it easy to break something)